reference/global-api-information.md

```diff
@@ -13,7 +13,9 @@ The following directives are supported:
  * `base_path`
  * `base_params`
 
-Below is a basic ApiDefinition that defines global info as well info for a specific version:
+In addition to the above which may be specified globally or on a version, there is a `version_with` directive that is only applicable on the global version to define what versioning "scheme" your application will use. By default it's set to `[:header, :params]`, meaning Praxis will look for either an  `X-Api-Version` request header *or* an `api_version` query parameter to determine the version of your API to use for processing the request. To narrow it to one or the other scheme, you can set it to just that specific option (i.e. set it to `:header` to just use the `X-Api-Version` header and ignore any `api_version` param). There is an additional scheme, `:path`, described later.
```

Comment:
Yeah. That's an awkward thing to describe. Especially the "below", since immediately below this paragraph is one that beings with another "Below" to preface the example.